vxtelegram/telegram.py

```diff
@@ -208,6 +183,44 @@ def add_status_bad_inbound(self, status_type, message, details):
             details=details,
         )
 
+    @inlineCallbacks
+    def handle_inbound_inline_query(self, message_id, inline_query):
+        # NOTE: Telegram supports multiple ways to answer inline queries with
+        #       rich content (articles, multimedia etc.) as opposed to simple
+        #       text messages.
+        #       see: https://core.telegram.org/bots/api#answerinlinequery
+        self.log.info(
+            'TelegramTransport receiving inline query from %s to %s' % (
+                inline_query['from']['id'], self.bot_username))
+
+        yield self.publish_message(
+            message_id=message_id,
+            content=inline_query['query'],
+            to_addr=self.bot_username,
+            from_addr=inline_query['from']['id'],
+            transport_type=self.transport_type,
+            transport_name=self.transport_name,
+            helper_metadata={
+                'type': 'inline_query',
+                'details': {
+                    'query_id': inline_query['id'],
+                },
```

Comment:
I'd like to propose a slightly different format for the `helper_metadata`, along the lines of:
```javascript
{
    'telegram': {
        'type': 'inline_query',
    }
}
```
Mostly this is just to match the structure of other existing `helper_metadata` uses, but I think it'll also help make clear that this is something specific to Telegram.